### ISSUE TITLE
Bug: Paragraph text fields not displaying pasted images

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#5340]
On TIA Scope Submission Details, when adding a scope comment, the system allows the user to paste images into the text field, and then the image disappears. Video demo [here](https://web.microsoftstream.com/video/d83afd2f-71d2-407f-8dee-ac0d96928ed4)

**Comment by ChrispinP** [comment]
Sent a message to Knack about the bug, awaiting response.

**Comment by ChrispinP** [comment]
Received Knack Response: Hey Chrispin! Happy to help with this. The behavior your seeing is intended. You won't be able to drag an image into a rich text view within the builder. Knack works a bit differently than many other services; we host your record data which might include images, but we don't host your images for rich text views. Your images must be hosted elsewhere on the web. You'll want to link them to your rich text views accordingly. You can read-up on how to do this on our Knowledge Base. Just reference this article and check out the section on Using Images.
As always, if you need any more help with this just reach back out. 😄

TLDR: **Images need to be publicly hosted and can only be added to Knack Rich Text fields with a URL**

Some Caveats:

- At this time, anchor links are not supported by rich text views.

- Broken images that are embedded with links may be due to the URL not being public. For example, images embedded from Google Drive will appear broken. We have had some customers resolve this issue by using conversion tools such as http://app.bytenbit.com/index.html to convert the image.

- If you're using a link copied from Dropbox, you'll need to remove the ?dl=0/ and add /?raw=1 to the end of your URL in order to insert the image itself, not the page where the image is hosted.

[Managing Rich Text Views](https://learn.knack.com/article/dzsvoi01z7-rich-text#using_images)

**Comment by ChrispinP** [comment]
no further action necessary, closing as complete
